Singer.API/ClientApp/src/app/modules/shared/components/generic-overview/generic-overview.component.ts

```diff
@@ -18,8 +20,8 @@ export abstract class GenericOverviewComponent<
 > implements AfterViewInit {
    @ViewChild(MatPaginator)
    paginator: MatPaginator;
-   @ViewChild(MatSort) sort: MatSort;
-   @ViewChild('filterInput')
+   @ViewChild(MatSort, { static: true }) sort: MatSort;
+   @ViewChild('filterInput', /* TODO: add static flag */ {})
```

Comment:
Why not, you know... just add it?

Reply:
Because it's auto generated. I didn't check all files for these TODOs. Will do now.